rest/blip_sync.go

```diff
@@ -63,7 +63,7 @@ func (h *handler) handleBLIPSync() error {
 	server := blipContext.WebSocketServer()
 	defaultHandler := server.Handler
 	server.Handler = func(conn *websocket.Conn) {
-		h.logStatus(http.StatusSwitchingProtocols, fmt.Sprintf("[%s] Upgraded to BLIP+WebSocket protocol%s", blipContext.ID, h.formattedEffectiveUserName()))
+		h.logStatus(http.StatusSwitchingProtocols, fmt.Sprintf("[%s] Upgraded to BLIP+WebSocket protocol which uses subprotocol %s%s", blipContext.ID, blipContext.ActiveSubprotocol(), h.formattedEffectiveUserName()))
```

Comment:
Can probably make this a bit more information-dense:

```suggestion
		h.logStatus(http.StatusSwitchingProtocols, fmt.Sprintf("[%s] Upgraded to WebSocket protocol %s+%s%s", blipContext.ID, blip.WebSocketSubProtocolPrefix, blipContext.ActiveSubprotocol(), h.formattedEffectiveUserName()))
```

```
2021-06-21T19:43:31.532+01:00 [INF] HTTP+: #004: --> 101 [742ca3ba] Upgraded to WebSocket protocol BLIP_3+CBMobile_3 (as <ud>user</ud>) (0.0 ms)
```